packages/main/test/pages/Popover.html

```diff
@@ -467,6 +474,19 @@
 			popNoFocusableContent.innerHTML = "Second button is clicked";
 			popNoFocusableContent.openBy(event.target);
 		});
+
+		btnOpenDynamic.addEventListener("click", function () {
+			var popover = document.createElement("ui5-popover"),
+				button = document.createElement("ui5-button");
+
+			button.innerText = "Focusable element";
```

Comment:
IE fix - button.textContent

Also remove all "(event) => { " functions from this file.

Reply:
done